src/resources/assets/scss/customs/_list.scss

```diff
@@ -0,0 +1,335 @@
+    top: 26px;
+    left: 115px;
+    height: calc(100% - 6px);
+    width: calc(100% - 30px);
+    z-index: 999;
+    border: none;
+    box-shadow: none;
+
+    &>img {
+      top: calc(50% - 16px);
+      left: calc(50% - 16px);
+      position: absolute;
+    }
+  }
+
+  .dataTables_length select {
+    min-width: 3.6rem;
+    margin-right: 0.4rem;
+  }
+
+  #crudTable {
+    margin-top: 0 !important;
+
+    &.dtr-inline.collapsed>tbody>tr>.dtr-control:before {
+      background-color: transparent;
+      color: #636161;
+      font-family: 'Line Awesome Free';
+      font-weight: 900;
+      width: 16px;
+      content: "\f142";
+      font-size: 21px;
+      box-shadow: none;
+      border: none;
+      display: inline;
+      position: relative;
+      top: 0;
+      left: 0;
+      margin: 0 0 0 -.25rem;
+    }
+
+    &.dtr-inline.collapsed>tbody>tr>.dtr-control:before {
+      top: 0;
+    }
+
+    td, th {
+      outline: none;
+      vertical-align: middle;
+    }
+
+    thead > tr {
+      &>th:not(.sorting_disabled),
+      &>td {
+        padding-right: 30px;
+      }
+    }
+
+    .sorting:before,
+    .sorting_asc:before,
+    .sorting_desc:before,
+    .sorting_asc_disabled:before,
+    .sorting_desc_disabled:before {
+      right: 0.4em;
+      top: 1em;
+      content: "\f0d8";
+      font: normal normal normal 14px/1 "Line Awesome Free";
+      font-weight: 900;
+    }
+
+    .sorting:after,
+    .sorting_asc:after,
+    .sorting_desc:after,
+    .sorting_asc_disabled:after,
+    .sorting_desc_disabled:after {
+      right: 0.4em;
+      content: "\f0d7";
+      font: normal normal normal 14px/1 "Line Awesome Free";
+      font-weight: 900;
+    }
+
+    tbody>tr:first-of-type>td:first-of-type,
+    tbody>tr:not(.shown)+tr>td:first-of-type,
+    thead>tr>th:first-of-type,
+    tfoot>tr>th:first-of-type {
+      padding: 0;
+      min-height: 1rem;
+    }
+
+    &.has-hidden-columns .details-control {
+      margin-left: .4rem;
+    }
+
+    &.has-hidden-columns {
+      thead tr th:first-child,
+      tfoot tr th:first-child {
+        padding-left: 1.35rem!important;
+      }
+    }
+
+    .details-control {
+      display: flex;
+    }
+
+    .crud_bulk_actions_checkbox {
+      display: flex;
+      margin: 0 0 0 .625rem;
+    }
+
+    .crud_bulk_actions_checkbox input {
+      width: 1rem;
+      height: 1rem;
+    }
+
+    &:not(.has-hidden-columns) {
+      tr td:first-child {
+        padding-top: 1.25rem!important;
+      }
+    }
+    tr th:first-child,
+    tr td:first-child {
+      display: flex;
+      align-items: center;
+      padding: 1rem 0 1rem 0.6rem!important;
+    }
+
+    &.has-details-row {
+      thead tr th:first-child,
+      tfoot tr th:first-child {
+        padding-left: 2.1rem!important;
+      }
+    }
+
+    &.has-details-row.has-hidden-columns {
+      thead tr th:first-child,
+      tfoot tr th:first-child {
+        padding-left: 2.725rem!important;
+      }
+    }
+

```

Comment:
In Articles CRUD I find that sometimes on the page it shows both the checkbox and the three dots, other times the checkbox is missing:

![Screenshot 2021-12-27 at 09 10 59](https://user-images.githubusercontent.com/1032474/147445108-014cab2b-0570-4062-b4b9-08cc7cf95d83.png)
![Screenshot 2021-12-27 at 09 11 07](https://user-images.githubusercontent.com/1032474/147445110-108cffa5-9a6e-444d-9410-287828d8d43c.png)



I couldn't pinpoint _when_ this is happening, I just browsed different pages and sometimes the same page works fine, sometimes it doesn't. Clearing the cache and views didn't seem to help.

Reply:
The same problem as above, by making the first column with high priority this issue is solved.